packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -355,15 +341,16 @@ class ChartController extends DisposableController
     }
   }
 
-  double xPositonToYCanvasCoord(double y) {
-    final zeroPosition = xCanvasChart + canvasChartWidth;
-    return zeroPosition - yScale?.tickFromValue(y);
-  }
+  double get zeroXPosition => xCanvasChart + canvasChartWidth;
```

Comment:
do we really need these to be relative to xCanvasChart now that we are using a translate?

Reply:
zeroXPosition is no longer used.